android/app/src/main/java/updated/mysterium/vpn/ui/pop/up/PopUpNoAmount.kt

```diff
@@ -0,0 +1,33 @@
+package updated.mysterium.vpn.ui.pop.up
+
+import android.view.LayoutInflater
+import androidx.appcompat.app.AlertDialog
+import network.mysterium.vpn.databinding.PopUpNoAmountBinding
+
+class PopUpNoAmount(layoutInflater: LayoutInflater) {
+
+    var bindingPopUp = PopUpNoAmountBinding.inflate(layoutInflater)
+    private var onTryAgainAction: (() -> Unit)? = null
+    private lateinit var dialog: AlertDialog
+
+    fun tryAgainAction(onTryAgainAction: () -> Unit) {
```

Comment:
I think we can get rid of this encapsulation level and just make `onTryAgainAction` property public and set right on property, without function

Reply:
Done